src/deviceApiCalls/transports/extension.transport.js

```diff
@@ -16,13 +18,21 @@ export class ExtensionTransport extends DeviceApiTransport {
 
     async send (deviceApiCall) {
         if (deviceApiCall instanceof GetRuntimeConfigurationCall) {
-            return deviceApiCall.result(await extensionSpecificRuntimeConfiguration(this.config))
+            return deviceApiCall.result(await extensionSpecificRuntimeConfiguration(this))
         }
 
         if (deviceApiCall instanceof GetAvailableInputTypesCall) {
             return deviceApiCall.result(await extensionSpecificGetAvailableInputTypes())
         }
 
+        if (deviceApiCall instanceof SetIncontextSignupDismissedAtCall) {
+            return deviceApiCall.result(await extensionSpecificSetIncontextSignupDismissedAt(deviceApiCall.params.value))
+        }
+
+        if (deviceApiCall instanceof GetIncontextSignupDismissedAtCall) {
+            return deviceApiCall.result(await extensionSpecificGetIncontextSignupDismissedAt())
+        }
```

Comment:
Device calls and scheams in for both of these -- let me know if there's anything missing